RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+14.1
+-----
+* Block Editor: Add support for Dark Mode
```

Comment:
Since this branch targets feature branch, it's probably not going to get released just yet :)

Reply:
Removed.